lib/jekyll-feed.rb

```diff
@@ -12,6 +12,14 @@ def config
       @context.registers[:site].config
     end
 
+    def path
+      if config["feed"] && config["feed"]["path"]
```

Comment:
Yeah, if the goal is only to support `feed` as a hash then blowing up might be a good thing. I don't know the conventions around this kind of thing in Jekyll-land which is why I raised it.